public/templates/index.html

```diff
@@ -28,7 +28,10 @@
           {{selectedHost == null ? 'All Systems' : selectedHost}}</button>
         <button type="button" id="date-picker-btn" class="time-option-select-btn btn flex-item-search">
           {{ isNullorEmpty(userDateTimeSeeked) ? 'Now' : userDateTimeSeeked }}</button>
-        <button type="button" id="settings-btn" class="settings-select-btn btn flex-item-search">Settings</button>
+        <!--COCO Begin-->
```

Comment:
By running this index.html can I check these changes or it is having any dependency?

Reply:
No deekshith you cant you have to build it and install it with kibana base image